metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -966,6 +971,7 @@ void deleteTopic(ControllerRequestContext context, Uuid id, List<ApiMessageAndVe
         }
         records.add(new ApiMessageAndVersion(new RemoveTopicRecord().
             setTopicId(id), (short) 0));
+        log.info("Successfully added RemoveTopicRecord for topic {} with ID {}", topic.name, id);
```

Comment:
I think what would be useful is logging the full result set. Like, each topic and the results of the deletion (success, or whatever error message we got.) If we make this a single log message at INFO it will be much more efficient.